roles-in-tech/product-manager.md

```diff
@@ -1,91 +1,112 @@
+- Know the skills Product Manager should have.
+- Decide whether they are suitable for a future as a Product Manager.
 
 ### Specific Things To Teach
 
-- Focus on this specific talking point
-- and this
-	- This is a sub-thing about the thing
-- Even more things about the things
-- Even more things about the things
+- What is Product Management?
+- What do Product Managers do?
+- Skills PM should have?
 
 ### Materials
 
 - [Techtonica Slides - Roles in Tech: Product Management](https://docs.google.com/presentation/d/1BxTU4nugas3Z30kIawpJtLhSBL6b5tF1P-d03Hm_JOI/edit?usp=sharing)
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
+- [ Who are PM? short video (7.53 min)](https://www.youtube.com/watch?v=2dczveSrsv8) - A short video by Ravijot Chugh which explains 
+who are Product Managers and what do they do?
+- [ Product Manager's Role (10 min)](https://www.altexsoft.com/blog/business/product-management-main-stages-and-product-manager-role/) 
+Use this guide to know more about role of Product Manager.
 
 ### Lesson
 
-Here's text about introducing something and how it works.
-
-- [Be sure to link the slideshow, video,](google.com)
-- [or other materials you expect them to follow as part of the lesson.](google.com)
-
-Build on the first information. Have apprentices guess things, do an activity, etc.
-
-Make sure to mention these things:
-
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
-
-
+   Go through the slide given in material section to get an idea about Product Management.
+
+ #### Product Management    
+  
+  Product management can be defined as the general business structure within a company that supports and 
+   manages all the activities related to developing, planning, forecasting, marketing and selling a product 
+   all through its lifecycle.
+	
+#### Job of Product Manager
+  
+    Product managers are the official translators within a company, helping commercial teams understand what 
+    the technical, engineering teams mean and vice-versa.
+    
+    Responsibilities that a product manager usually takes are :-
+    - User/market research and analysis
+    - Create a solid Product strategy
+    - Learn to prioritize and re-prioritize various work threads.
+    - Create a product and feature roadmap.
+    - Know how to make a product stable and scalable.
+    - Communicate effectively with Designers, Customers, Engineers and clients.  
+    
+   The Role of PM may change depending on type of product and the company stage, etc.
+
+#### Qualities of PM:-
+    - Strategic thinker
```

Comment:
Same as line 60

Reply:
Can you please check the changes I have made. I wasn't clear what you meant by 'Same as line 60'

Reply:
@alodahl meant to say that the meaning of the line 69 and 60 are same. So you could remove one of them. @raibove

Reply:
No, I mean this has the same problem that I mentioned for line 60.

Reply:
It is in code notation but it is not code -please change markdown

Reply:
Oh.

Reply:
@raibove This section and the section starting at line 60 are gray when seen in the markdown preview, and they should not be gray unless it is code.  Please fix.